oauthproxy.go

```diff
@@ -745,12 +745,11 @@ func (p *OAuthProxy) UserInfo(rw http.ResponseWriter, req *http.Request) {
 
 // SignOut sends a response to clear the authentication cookie
 func (p *OAuthProxy) SignOut(rw http.ResponseWriter, req *http.Request) {
-        //<!--COCO Begin-->
+	//<!--COCO Begin-->
 	p.ClearSessionCookie(rw, req)
-        // COCO: calling CocoLogout to call coco logout API
-        p.CocostoreLogout(rw, "logging out")
-        //<!--COCO End-->
-
+	// COCO: calling CocoLogout to call coco logout API
+        p.CocoLogout(rw, "Signing out")
```

Comment:
- p.CocoLogout is not aligned. Kindly do self-review before you submit the code

Reply:
handled